GameProject/Assets/TurnBasedAssets/Scripts/MouseSelection.cs

```diff
@@ -0,0 +1,56 @@
+using UnityEngine;
+
+namespace TurnBasedAssets.Scripts
+{
+    public class MouseSelection : MonoBehaviour
+    {
+        private ISelection _selection;
+        [SerializeField] GameObject cube;
+        [SerializeField] float distanceY;
+        [SerializeField] float offset;
+        [SerializeField] float gridSize;
+        [SerializeField] Camera camera;
+        private Plane _plane;
+        private Vector3 _distanceFromCamera;
```

Comment:
This doesn't match your other private fields within the project. Please choose either `_camelCase` or `camelCase`.

Reply:
This is still a problem in all affected files in this commit. Please resolve.